rollup.config.js

```diff
@@ -48,9 +51,21 @@ export default {
 		}),
 
 		svelte({
+			dev: !production,
+			preprocess: autoPreprocess(),
 			compilerOptions: {
 				// enable run-time checks when not in production
 				dev: !production
+			},
```

Comment:
Shouldn't we add that we are making changes on hmr in the PR description?

Reply:
hmr is already upstreamed.